Nette 3.0 compatibility

andrejsoucek opened the pull request:
- Updated dependencies
- Updated code to match Nette 3.0
- Updated tests
Check the onMatch deletion, please.

Commit:
Nette 3.0 compatibility

paveljanda commented:
Thank you, I will check that

paveljanda commented:
Thanks for the PR!

The `::onMach` event is needed though. (https://github.com/contributte/api-docu/blob/master/src/Starter.php#L95)
Could you please put it back?

andrejsoucek commented:
Will do. So it is used in the other project to generate documentation?

paveljanda commented on src/ApiRoute.php:
```diff
@@ -11,18 +11,16 @@
 use Nette\Utils\Strings;
 
 /**
- * @method mixed onMatch(static, Nette\Application\Request $request)
```
Could you please also keep this doc block? Thanks!

paveljanda commented:
@andrejsoucek Exactly. This project works hand in hand with `api-docu` - documentation generating vendor. Thanks.

f3l1x commented:
Please, take a look at our bare repo.

https://github.com/contributte/contributte/tree/master/bare

andrejsoucek commented:
any progress?

matlad committed:
FIX NotWorkingAnnotations
- Remove run tag is deprecated replace by addicting getService into initialize throw afterCompile
- Fixing loading presenters by tag 'nette.presenter' tag no longer exist replaces by finding by type of IPresenter

matlad committed:
remove unused import

matlad committed:
removing unused import

matlad committed:
Actualization of composer.lock for nette 3.0

matlad committed:
Format Nette code style

matlad committed:
fix class name

mabar committed:
Add master alias

paveljanda commented:
@andrejsoucek Can you please resolve the conflicts? Thanks

andrejsoucek committed:
Merge branch 'master' into nette-3.0

andrejsoucek commented:
should be ok now

paveljanda commented:
Thanks!
